src/main/java/com/digitax/payload/ApiRes.java

```diff
@@ -5,31 +5,25 @@
 import lombok.Setter;
 import lombok.experimental.Accessors;
 
-import java.util.Map;
-
 @Getter
 @Setter
 @Accessors(chain = true)
 public class ApiRes<T>{
-    private ResCode co;
-    private Map<?, ?> extras;
-    private T da;
+    private int code;
+//    private Map<?, ?> extras;
```

Comment:
@Jayanta-Redapple You can remove commented out code

Reply:
Sure I will remove the commented code